src/components/Error404.tsx

```diff
@@ -0,0 +1,27 @@
+import Menu from './Menu';
+
+let Error404 = () => {
+
+    return (
+        <div>
+            <Menu />
```

Comment:
I'd remove `Menu` from this component and actually place it in the `App.tsx` file if you can, that way you won't need to explicitly include this component in every page that has the `Menu` component above it.